packages/builder/src/builder.js

```diff
@@ -542,6 +544,20 @@ export default class Builder {
     )
   }
 
+  resolvePluginsMode() {
```

Comment:
The same code in `resolvePlugins` should be removed.
And this method could be async and wrapped with Promise.all.

Reply:
Oh, yes, it should be removed, it was in fact, but then i debugged and forgot to remove again :)
@clarkdo why async? It just iterate over plugins with map

Reply:
All the resolving can be in parallel for better performance